src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/resource/OffenderController.kt

```diff
@@ -23,9 +23,9 @@ import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.service.ComService
 import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.service.OffenderService
 
 @RestController
+@Tag(name = Tags.OFFENDER)
```

Comment:
All endpoints in this controller inherit the Offender Tag

This effectively adds a tag to the missing endpoint